### DISCUSSION TITLE
Where do hidden tasks go?

### DISCUSSION THREAD

**Discussion by practicalwellbeing** [stacks-task-manager/stacks2#47] [General]
This might be a dumb question.

If you 'auto-hide' tasks where do they go and how do you get them back?

I've disabled auto-hide in the general sessions so I'm good from now on, but my previous done tasks can't be seen.

**Comment by synapse** [comment]
Hey @practicalwellbeing, you need to use the filter (top-right corner) from the project toolbar and set `All tasks` from the `Task state` drop-down menu

**Reply by practicalwellbeing** [reply]
Thanks, I'll do that.
